src/app/import/components/modal-upload/modal-upload.component.html

```diff
@@ -8,12 +8,37 @@ <h1 class="flex flex-grow justify-center text-lg text-gray-900 font-bold">{{titl
     <div class="text-sm text-gray-500 mb-4 text-center">
         {{subTitle}}
     </div>
+    <div class="form-control" *ngIf="uploadFileType == UploadFileType.PROJECT">
+        <label class="text-gray-500 text-sm font-normal">Project title<em>- optional</em></label>
+        <input [value]="projectName" (input)="setProjectName($event.target.value)" type="text"
+            placeholder="Enter some title here ..." required autocomplete="off"
+            class="input input-sm input-bordered w-full placeholder-italic">
+        <div *ngIf="isProjectTitleDuplicate" class="text-red-700 text-xs mt-2">Project title exists</div>
+        <div class="flex flex-row mt-2">
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon icon-tabler icon-tabler-alert-triangle text-yellow-700"
+                width="20" height="20" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" fill="none"
+                stroke-linecap="round" stroke-linejoin="round">
+                <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+                <path
+                    d="M10.24 3.957l-8.422 14.06a1.989 1.989 0 0 0 1.7 2.983h16.845a1.989 1.989 0 0 0 1.7 -2.983l-8.423 -14.06a1.989 1.989 0 0 0 -3.4 0z" />
+                <path d="M12 9v4" />
+                <path d="M12 17h.01" />
+            </svg>
+            <label class="text-yellow-700 text-xs italic ml-2">If a project title is not specified, the project title
+                from
+                the
+                file
+                will be used, potentially leading to duplicate project names.</label>
```

Comment:
I am not 100% sure if this is better, but still I would rephrase this to something like:

_If no project title is specified, the title mentioned in the uploaded data will be used, potentially leading to duplication of project titles._

Reply:
Yes, this looks good.